common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1509,6 +1510,14 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("300s")
 
+  val HDFS_REMNANTDIRS_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.master.hdfs.remnantDirs.timeout")
```

Comment:
The storage namespace should be used, and we'd better emphasize it applies to app level in the name
```suggestion
    buildConf("celeborn.master.storage.hdfs.appDirs.expireDuration")
```

Reply:
Check the ding-ding group, I explained the reason why I choose this name.

Reply:
@AngersZhuuuu do you have suggestion for its name?